core/primitives/src/epoch_manager.rs

```diff
@@ -139,6 +149,19 @@ impl AllEpochConfig {
             config.chunk_producer_kickout_threshold = 80;
             config.validator_selection_config.num_chunk_only_producer_seats = 200;
         }
+
+        if chain_id == "testnet"
+            && checked_feature!("stable", TestnetMoreChunkOnlyProducers, protocol_version)
+        {
+            let num_shards = config.shard_layout.num_shards() as usize;
+            // On testnet, genesis config set num_block_producer_seats to 200
+            // This is to bring it back to 100 to be the same as on mainnet
```

Comment:
not 💯anymore

This change will decrease the total number of validators as well. Maybe increase the number of cop to keep it high enough so that more people can join in as validators? I don't feel strongly about it either way, just brainstorming.

Reply:
> not 100 anymore

fixed

> will decrease the total number of validators

Indeed, but with the current number of testnet validators it doesn't seem to be an issue.
Will increase the number of chunk producers if somebody has a strong opinion in favor of that.